example/ios/KlaviyoReactNativeSdkExample/AppDelegate.mm

```diff
@@ -90,12 +92,11 @@ - (void)userNotificationCenter:(UNUserNotificationCenter *)center
 
 // Installation Step 13: Implement this method to receive deep link. There are some addition setup steps needed as mentioned in the readme here -
 // https://github.com/klaviyo/klaviyo-swift-sdk?tab=readme-ov-file#deep-linking
-// additionally routing to the right screen in the app based on the url is also something that should be handled
+// Calling `RCTLinkingManager` is required for your react native listeners to be called
 - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options {
-  return [PushNotificationsHelper handleDeepLinksWithUrl:url];
```

Comment:
What did this do?

Reply:
ahh

Reply:
I was just handling like we'd do on iOS. Now it will route to RN where customer can decide how to navigate to their pages.